src/Files.App/ViewModels/UserControls/SidebarViewModel.cs

```diff
@@ -114,7 +114,7 @@ public void UpdateSidebarSelectedItemFromArgs(string? arg)
 
 			item = filteredItems.FirstOrDefault(x => x.Path.Equals(value, StringComparison.OrdinalIgnoreCase));
 			item ??= filteredItems.FirstOrDefault(x => x.Path.Equals(value + "\\", StringComparison.OrdinalIgnoreCase));
```

Comment:
I think this line is no longer necessary.

Reply:
You're right, I removed the line and tested it again